### PR TITLE
Abu

### PR DESCRIPTION
**Pull request by aaabuk02**
Refactored cart to start using redux and updated catalog data.

### CODE REVIEW

File: src/Components/CartModal/CartModal.js [hunk 1]
```diff
@@ -1,26 +1,29 @@
 import Button from "../UI/Button/Button";
 import Modal from "react-bootstrap/Modal";
 import "../ConfirmModal/ConfirmModal.scss"
-const CartModal = ({ show, handleRemove, handleClose, props }) => {
-  let propTotal = 0;
-  for (let i = 0; i < props.length; i++) {
-    propTotal += props[i].event_pricesFrom;
-  }
+import { useSelector } from 'react-redux'
+import { useDispatch } from 'react-redux'
```
Comment (DanRossLi) [inline src/Components/CartModal/CartModal.js:5]: You can import both on the same line, use a comma between the two

File: src/Components/CartModal/CartModal.js [hunk 2]
```diff
@@ -1,26 +1,29 @@
...
   return (
     <>
       <Modal show={show} dialogClassName="modal-content-style" onHide={handleClose} backdrop="static" keyboard={false} centered>
         <Modal.Header closeButton>
           <span>Cart</span>
         </Modal.Header>
         <Modal.Body>
-          <ul>{props.map((product, index) => <li key={index} onClick={() => handleRemove(product)}>{product.event_title}, {product.event_pricesFrom}</li>)}</ul>
-          <p>Total: ${propTotal}</p>
+          <ul>{cart.cartEvents?.map((product, index) => <li key={index} onClick={() => dispatch(removeFromCart((index)))}>{product.event_title}, {product.event_pricesFrom}</li>)}</ul>
```
Comment (DanRossLi) [inline src/Components/CartModal/CartModal.js:19]: Nice work. Could you break to new lines to make more readings

Comment (DanRossLi) [reply src/Components/CartModal/CartModal.js:19]: Readible

**Review by DanRossLi — COMMENTED** [review]
Looks good

**Review by StevenJohnsonPersonal — APPROVED** [review]

**Review by Lnguyen1996 — APPROVED** [review]
Awesome!

**Review by DoctorGo — COMMENTED** [review]
looks good

**Pull request merged**